### DISCUSSION TITLE
State of accessibility?

### DISCUSSION THREAD

**Discussion by tscislo** [xyflow/xyflow#2675] [Q&A]
What is the general state of accessibility for react flow?
According to the page: https://reactflow.dev/docs/guides/accessibility/ it should be fully accessible. What I mean by that is that nodes and edges should be focusable with keyboard navigation.
However currently with each demo that I use for example this: https://codesandbox.io/s/uv695d?file=%2FApp.js&utm_medium=sandpack and I try to tab into workflow the only part that is focusable is Control component. Edges and nodes are not focusable.
The only thing that I was able to do with keyboard is move node when I previously select it with mouse.
Do I miss something here?

**Comment by tscislo** [comment]
I even set forcefully 
```
        nodesFocusable={true}
        nodesDraggable={true}
```
which according to the doc should be default

**Reply by moklick** [reply]
The custom nodes in that example are not styled differently when they are focused. Better try this default example:
https://j8lkh.csb.app/

Tab to get to another node or edge and then select via Space or Enter then you move them with the arrow keys.

**Reply by tscislo** [reply]
Thank you, yes you're right! When we apply appropriate styling for :focus-visible and .selected it works nicely. Thank you!
